react/react1/week3/src/components/ListItem.css

```diff
@@ -0,0 +1,63 @@
+li {
+  font-weight: 700;
+}
+
+.done {
+  text-decoration: line-through;
+  color: #6666;
+}
```

Comment:
You have a lot of copied code, you probably should change templates that you use for practice